Makefile

```diff
@@ -1,9 +1,12 @@
-.PHONY: test nits docs nightly
+.PHONY: test nits docs nightly test-clean
 
 all: test nits docs
 
 test:
-	cargo test --release
+	cargo insta test --unreferenced=reject
+
+test-clean:
```

Comment:
`test` and `test-clean` do the same thing?

Reply:
No, test clean deletes all of the unreferenced snapshot files. Useful if you are working on the testing infra